pkg/sql/codegen/feature/codegen_feature_column.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 The SQLFlow Authors. All rights reserved.
```

Comment:
Should we move package `feature` under `pkg/step/feature/` or keep it here? Seems we already have a package named `feature` there.

Reply:
Thanks for recommendation. Moved to `codegen`.